Submissions/bob-karemera-shema.json

```diff
@@ -0,0 +1,16 @@
+{
+    "name": "Bob Karemera Shema",
+    "img": "https://avatars.githubusercontent.com/u/72112797?v=4",
+    "email": "bobshema14@gmail.com",
+    "links": {
+      "website": "portfolio-7pg8.onrender.com",
+      "linkedin": "https://www.linkedin.com/in/bob-karemera-shema",
+      "github": "https://github.com/Bob-Karemera-Shema"
+    },
+    "jobTitle": "Software Engineer",
+    "location": {
+      "city": "Cambridge",
+      "state": "[YOUR_STATE_OR_PROVINCE]",
```

Comment:
Can you please remove this placeholder? @Bob-Karemera-Shema 

The instructions are in the [README](https://github.com/zero-to-mastery/ZtM-Job-Board#how-to-add-your-name-to-the-list).